### PR TITLE
Add support for network v1

### LINKED ISSUE #638: ACS Deployment fails on Kubernetes version 1.22
Hi,
Deployment on Kubernetes v1.22 fails because networking.k8s.io/v1beta1 is depricated. New version only supports networking.k8s.io/v1.
Is this update in pipeline ? 

Regards

### PR DESCRIPTION
**Pull request by hlavki**
This pull request fixes #638 
Running on kubernetes >= 1.22 where apiVersion `networking.k8s.io/v1beta1` was removed.

Added this MR because I think #653 according to [documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#examples) does not work because of `pathType = Exact`. This MR uses `pathType = Prefix`.

### CODE REVIEW

File: helm/alfresco-content-services/templates/ingress-repository.yaml [hunk 1]
```diff
@@ -50,10 +52,28 @@ spec:
   {{- end }}
       paths:
       - path: {{ .Values.repository.ingress.path }}
+        {{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion }}
+        pathType: Prefix
+        backend:
+          service:
+            Name: {{ template "content-services.shortname" . }}-repository
```
Comment (sebastiankomander) [inline helm/alfresco-content-services/templates/ingress-repository.yaml:59]: name must be lower-case!

**Comment by mteodori** [comment]
duplicates #653

File: helm/alfresco-content-services/templates/ingress-repository.yaml (hunk 1 above)
Comment (hlavki) [reply helm/alfresco-content-services/templates/ingress-repository.yaml:59]: Yes, this PR contains 2 commits and it's already fixed in the second commit.

**Comment by hlavki** [comment]
@mteodori, kubernetes [documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types) says:
> Each path in an Ingress is required to have a corresponding path type. 

but `pathType` is missing in #652.

**Comment by mteodori** [comment]
closing in favour of #652 please add comments or requests there

**Pull request closed without merging**